cmd/podman/spec.go

```diff
@@ -652,6 +652,28 @@ func (c *createConfig) GetContainerCreateOptions() ([]libpod.CtrCreateOption, er
 		}
 	}
 
+	if len(c.Volumes) != 0 {
+		// Volumes consist of multiple, comma-delineated fields
+		// The image spec only includes one part of that, so drop the
+		// others, if they are included
+		volumes := make([]string, 0, len(c.Volumes))
+		for _, vol := range c.Volumes {
+			splitVol := strings.Split(vol, ":")
```

Comment:
This would be slightly more efficient with [`strings.SplitN`][1].  And because you know there will always be at least one entry in the result, you can drop the temporary variable and use:

```go
volumes = append(volumes, strings.SplitN(vol, ":", 2)[0])
```

[1]: https://golang.org/pkg/strings/#SplitN

Reply:
Good call, fixed